test/test-capablities.js

```diff
@@ -0,0 +1,30 @@
+var capabilities = require('../lib/capabilities');
+var async = require('async');
+var tap = require('tap');
+
+tap.test('targetctl capabilities', function(t) {
+  var caps = ['tracing', 'metrics', 'watchdog',
+      'cpuprofile', 'heapsnapshot', 'patch'];
+  var list = capabilities.list();
+  t.ok(list, 'Capabilities are defined.');
+  if (!list) {
+    t.end();
+  } else {
+    async.mapSeries(caps,
```

Comment:
This isn't necessary, `t.test()` will do this for you if you make each feature a sub-test.